docs/src/developing/clients/javascript-reference.md

```diff
@@ -0,0 +1,873 @@
+---
+title: Web3 API Reference
+---
+
+## Web3 API Reference Guide
+
+### Authorized
+
+[Source Documentation](https://solana-labs.github.io/solana-web3.js/classes/Authorized.html)
+
+Authorized is an object used when creating an authorized account for staking within Solana. You can designate a `staker` and `withdrawer` separately, allowing for a different account to withdraw other than the staker.
+
+You can find more usage of the `Authorized` object under [`StakeProgram`](javascript-api.md#StakeProgram)
+
+### BpfLoader
```

Comment:
Let's leave this out, I'd like to deprecate the bpf loader from web3 and encourage devs to use the cli instead